### PR TITLE
Updated login page

### PR DESCRIPTION
**Pull request by MigleMic**
Updated login page visually, added background

### CODE REVIEW

File: Encounter1/MainActivity.cs [hunk 1]
```diff
@@ -2,28 +2,53 @@
...
+
+            //LinearLayout layout = (LinearLayout)FindViewById(Resource.Id.linearLayout1);
+           // layout.SetBackgroundResource(Resource.Drawable.background);
+
+            animationDrawable = (Android.Graphics.Drawables.AnimationDrawable)Resources.GetDrawable(Resource.Drawable.background);
+            LinearLayout img = (LinearLayout)FindViewById(Resource.Id.linearLayout1);
+            img.SetBackground(animationDrawable);
+
+            //ImageView img = (ImageView)FindViewById(Resource.Id.linearLayout1);
+            //img.SetImageDrawable((Android.Graphics.Drawables.Drawable)animationDrawable);
```
Comment (Waltzerizm) [inline Encounter1/MainActivity.cs:41]: Will these be used in the future?

**Review by Waltzerizm — APPROVED** [review]
Overall looks good, I liked the transitions:)

File: Encounter1.sln [hunk 2]
```diff
@@ -7,16 +7,104 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Encounter1", "Encounter1\En
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Ad-Hoc|Any CPU = Ad-Hoc|Any CPU
```
Comment (Mdominykas) [inline Encounter1.sln:10]: Why are all these lines added to Encounter1.sln file?

File: Encounter1/MainActivity.cs (hunk 1 above)
Comment (MigleMic) [reply Encounter1/MainActivity.cs:41]: No, we will not need them. I forgot to delete them. Thanks for showing me

File: Encounter1.sln (hunk 2 above)
Comment (MigleMic) [reply Encounter1.sln:10]: I was trying to add iOS to our app but it didn't work out for me. 
I will remove these lines quickly

**Review by Mdominykas — APPROVED** [review]
the colours look very nice.

**Comment by hickoryeyes** [comment]
~~Wow amazing~~ It looks so much better right now, great work!

**Review by hickoryeyes — COMMENTED** [review]
~~Wow amazing~~ It looks so much better right now, great work!

**Review by hickoryeyes — APPROVED** [review]

**Pull request merged**